.github/workflows/tests.yml

```
@@ -8,7 +8,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [3.6, 3.7, 3.8, 3.9]
+        python-version: [3.6, 3.7, 3.8, 3.9, 3.10]
```

```suggestion
        python-version: ["3.6", "3.7", "3.8", "3.9", "3.10"]
```

Otherwise 3.10 means 3.1, because YAML's great until it isn't. :wink: